book/src/users/requirements.md

```diff
@@ -0,0 +1,24 @@
+# Requirements
+
+## Hardware
+
+Suitable:
+- Processor: x86 or Arm based. Minimum spec TBD.
+- RAM: Minimum TBD.
+- Disk: Any.
+
+Testing and reports of performance on the following are welcome:
+- RISC-V based processor.
+- Resource constrained (CPU/RAM)
+
+
+## Software
+
+- Unix based operating system
+- Rust installation (minimum `v1.65`)
+
+## Network
+
+Testing/reports of low-bandwidth network are welcome.
+
+Compatible with VPN use.
```

Comment:
I'm not sure this is accurate. I typically turn off my vpn when running a node, it introduces connectivity problems. - https://github.com/ethereum/trin/blob/dba58737467a8f8778c234d2ade81aa0ade3cce4/docs/getting_started.md#building-testing-and-running

Reply:
What sort of problem do you see? As i understand it, a can learn about its own external IP from a peer on handshake. After this handshake, the node can update its ENR IP. I made a diagram of my understanding of this in the NAT section that I think applies here.

Reply:
In the Part 1 NAT diagram below, the peer (bootnode) responds by telling Alice what it sees her IP address as. 
This should apply to a VPN, where the peer will see the VPNs IP address and can inform Alice (who updates her ENR).

`book/developers/core_concepts/finding_peers.md`. 
```mermaid
sequenceDiagram
    Alice IP1 PORT1-->>NAT IP2 PORT2: Hello bootnode (ENR with no IP)
    Note right of NAT IP2 PORT2: Stores Bootnode
    NAT IP2 PORT2-->>NAT IP2 PORT2: Maps from internal IP
    NAT IP2 PORT2-->>Bootnode: Hello bootnode (ENR with no IP)
    Bootnode-->>NAT IP2 PORT2: Hi, I notice your address is <IP2>:<PORT2>
    NAT IP2 PORT2-->>NAT IP2 PORT2: Maps to internal IP
    NAT IP2 PORT2-->>Alice IP1 PORT1: Hi, I notice your address is <IP2>:<PORT2>
    Alice IP1 PORT1-->>Alice IP1 PORT1: Updates ENR (<IP2>:<PORT2>)
    Alice IP1 PORT1-->>NAT IP2 PORT2: Thanks bootnode (ENR with <IP2>:<PORT2>)
    NAT IP2 PORT2-->>Bootnode: Thanks boodnode (ENR with <IP2>:<PORT2>)
    Note right of Alice IP1 PORT1: ... Hmm no strangers. Must be a NAT.

```

Reply:
Tbh, I believe you're right in theory, but in practice I've constantly had connection problems when my vpn is turned on. iirc others have had the same experience. Haven't sussed out what the cause of the problem is exactly, so that's why we recommend running trin without a vpn for now (as the old docs stipulated).

Imo, I think we should change the new documentation to reflect this. Reason being, I'd hate for a newcomer to the project to be discouraged b/c of this. I would update this comment to be something like `Trin should be compatible with VPN use, but if you experience difficulty connecting to the network we recommend disabling your VPN.` and add a similar clause to the other vpn doc comment... `Yes, you can use a Virtual Private Network (VPN), but if you encounter difficulty connecting to the Portal Network, we recommend disabling your VPN.`

I'm tracking this in #581 - I would like to resolve this, but tbh it's not at the top of my priority list, so I'm not sure how soon I'll be able to dig into this.

Reply:
Yes good compromise, I've updated both notes